fl4health/servers/nnunet_server.py

```diff
@@ -187,11 +183,11 @@ def initialize(self, server_round: int, timeout: Optional[float] = None) -> None
                 "Requesting initialization of global nnunet plans from one random client via get_properties",
             )
             random_client = self._client_manager.sample(1)[0]
-            ins = GetPropertiesIns(config=config)
-            properties_res = random_client.get_properties(ins=ins, timeout=timeout, group_id=server_round)
+            ins = GetPropertiesIns(config=self.fl_config | {"current_server_round": 0})
```

Comment:
The fl_config will have everything needed for this except for what server round we're currently on. So we just add it here. 0 is because no rounds have actually been done.

Reply:
Cool! Didn't know about | operator for dicts!

Reply:
Just be careful because `{}` will evaluate to `True` in this case. It's fine if that's not a concern.

Reply:
I maybe misreading your comment Marcelo, but in this case, the pipe isn't a binary operator but is rather a merge operator for the two dictionaries. So I think we're okay, but perhaps I'm missing something 🙂